todo_app/tests/test_unit.py

```diff
@@ -0,0 +1,16 @@
+from todo_app.todo import Item
+from todo_app.view_model import ViewModel
+
+def test_todo_items():
+    items_to_do = [Item(2344,"Answer emails", "To do"), Item(2345,"Do exercise", "In Progress") , Item(2346,"Go Home", "Done")]
+    VM = ViewModel(items_to_do)
+    ToDoItems = VM.todo_items
+    assert len (ToDoItems) == 1
+    assert ToDoItems[0].name == "Answer emails"
```

Comment:
This test is good, it checks that you get the correct item out, but note that you could have done a thorough comparison in a single line. E.g. `assert ToDoItems == [items_to_do[0]]` - assert that the resulting list is a list containing a single item which is the first item from the items_to_do variable.